### ISSUE TITLE
Keyboard navigation in notifications area results in tabbing over redudant links

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#11403]
**Describe the bug**

When using keyboard navigation or a screen reader, image links in notification cards (avatars of orgs and users) can be tabbed too and are in the accessibilty tree. They are redundant links for a keyboard user/screen reader because there are already textual links for them in a notification card.

**To Reproduce**

1. Go to https://dev.to/notifications
2. Start tabbing with the keyboard. You will land on the avatar of an author or organization.
3. If you press ENTER on either of these avatars it will navigate to the author's or organization's profile (expected).
4. However, if you keep tabbing in the notification card, you will land on the same links in textual form for the author and organization.
5. This all works fine, but it means more tabbing for a keyboard user because of the redundant links and the same thing would occur with a screen reader.

**Expected behavior**

The avatar links in a notifications card are removed from the accessibility tree and they can not be tabbed to via the keyboard.

**Screenshots**

![Nov-13-2020 09-26-24](https://user-images.githubusercontent.com/833231/99083586-bd394780-2593-11eb-92c6-6b2fe59600e9.gif)

**Desktop (please complete the following information):**

- OS, version: n/a
- Browser, version: n/a

**Smartphone (please complete the following information):**

- Device: n/a
- OS, version: n/a
- Browser, version: n/a

**Additional context**

n/a

**Comment by eevajonnapanula** [comment]
I'd like to work on this

**Comment by nickytonline** [comment]
Awesome @eevajonnapanula. Assigning this issue to you.

**Comment by nickytonline** [comment]
A `tabIndex="-1"` and a `aria-hidden="true"` on the avatars for orgs/users should do the trick or do you recommend a better approach @marcysutton?

**Comment by rhymes** [comment]
The latest Firefox beta, in the accessibility panel, displays tabbing order, if that can help:

<img width="1789" alt="Screenshot 2020-11-23 at 9 57 41 AM" src="https://user-images.githubusercontent.com/146201/99943921-5b869380-2d72-11eb-889c-5cb971a34b60.png">

**Comment by eevajonnapanula** [comment]
Alright, thanks! 

A question - how about notifications about more than two followers should work? How the card currently works is that if there are three or more followers, it shows the avatars, but the text contains only the name of the first follower on the list. So in that case, maybe it would be good to leave the avatars to the tab order and accessibility tree, as sighted/mouse-users can reach them, and other names than the first one are not in the text? This would still add one extra tab, but at the same time it wouldn't exclude non-sighted/keyboard users.

**Comment by eevajonnapanula** [comment]
I realised that I forgot a ping from my question, and as @forem/oss doesn't seem to work, pinging @nickytonline

**Comment by nickytonline** [comment]
> Alright, thanks! 
> 
> 
> 
> A question - how about notifications about more than two followers should work? How the card currently works is that if there are three or more followers, it shows the avatars, but the text contains only the name of the first follower on the list. So in that case, maybe it would be good to leave the avatars to the tab order and accessibility tree, as sighted/mouse-users can reach them, and other names than the first one are not in the text? This would still add one extra tab, but at the same time it wouldn't exclude non-sighted/keyboard users. 

@eevajonnapanula, in that particular case perhaps exclude the 1st avatar (if possible) so it only uses the text for the first follower to avoid that extra tab and this works still with fine for mouse users.

Use your best judgement as you continue through this and we can make small adjustments in the pull request if necessary.

Thanks again for taking this on!

**Comment by marcysutton** [comment]
To answer your question about skipping links @nickytonline, `tabindex="-1"` and `aria-hidden="true"` should be sufficient. I don't have sufficient data to test it myself, but it looks like you've landed on an acceptable solution. Great work, @eevajonnapanula!
